api/parsers/slots.go

```diff
@@ -0,0 +1,53 @@
+package parsers
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/hermeznetwork/hermez-node/common"
+	"github.com/hermeznetwork/hermez-node/db/historydb"
+	"github.com/hermeznetwork/tracerr"
+)
+
+type slotFilter struct {
+	SlotNum *uint `uri:"slotNum" binding:"required"`
+}
+
+// ParseSlotFilter func to parse slot filter from uri to the slot number
+func ParseSlotFilter(c *gin.Context) (*uint, error) {
+	var slotFilter slotFilter
+	if err := c.ShouldBindUri(&slotFilter); err != nil {
+		return nil, tracerr.Wrap(err)
+	}
+	return slotFilter.SlotNum, nil
+}
+
+type slotsFilters struct {
```

Comment:
Maybe would be a nice idea left those structs within the parsers public to allow a third-party library to reuse them.